src/main/java/org/graylog/plugins/usagestatistics/collectors/NodeCollector.java

```diff
@@ -108,10 +108,20 @@ private NodeInfo buildNodeInfo() {
         return NodeInfo.create(
                 NodeRole.fromCapabilities(serverStatus),
                 Version.CURRENT_CLASSPATH.toString(),
-                buildPluginInfo(plugins)
+                buildPluginInfo(plugins),
+                buildOsInfo()
         );
     }
 
+    private Os buildOsInfo() {
+        org.jsoftbiz.utils.OS os = org.jsoftbiz.utils.OS.getOs();
```

Comment:
Not sure if we want to wrap this in a try-catch block. I have no idea yet what happens on unknown operating systems.